notebooks/visualize/visualize.ipynb

```diff
@@ -55,7 +55,7 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.        value='OfficeActivity',
</pre><p>You might want to leave this as sorted(tables)[0]. There is a chance that someone might not have the OfficeActivity table in their workspace.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/mstic-dataexpl/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>